canonical-web-and-design/maas-ui
# feat(ui): update the logo

## Done

- Update to the new logo.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the full client.
- Check that the new logo appears when the app is loading.
- Visit a React page and the new logo should appear.
- Visit controller details and the new logo should appear.

## Fixes

Fixes: canonical-web-and-design/app-tribe#832.
